ami/install.bash

```diff
@@ -12,7 +12,7 @@ echo "deb https://apt.dockerproject.org/repo debian-stretch main" | tee -a /etc/
 apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
 
 # Add Mesos repo.
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E56151BF && \
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv DF7D54CBE56151BF && \
```

Comment:
did you create new AMI after thhis change?

Reply:
`DF7D54CBE56151BF` is the 64-bit fingerprint for our signing key.

I don't think it is urgent to create a new AMI right now.